context/app/api/client.py

```diff
@@ -211,7 +211,17 @@ def get_vitessce_conf_cells_and_lifted_uuid(self, entity, marker=None, wrap_erro
                 def get_assaytype(entity):
                     uuid = entity.get('uuid')
                     url = f"{current_app.config['SOFT_ASSAY_ENDPOINT']}/{uuid}"
-                    return requests.get(url, headers=self._get_headers()).json()
+                    headers = self._get_headers()
+                    try:
+                        request = requests.get(url, headers=headers)
+                        return request.json()
+                    except Exception as e:
```

Comment:
The exception is raised when the response doesn't contain json. Should there be a check for the response status?

Reply:
I figured we could implicitly handle any errors thrown by `requests.get` by including it in the try/except and catching all `Exception`s, not just the `JSONDecodeError` we were encountering before: https://docs.python-requests.org/en/latest/_modules/requests/exceptions

I have added `status` to the logged error, and also updated the error to be logged in the form of an object to make it easier to identify fields.